doc/dev/doc/failover.md

```diff
@@ -42,7 +42,7 @@ Let's assume both clusters are down, and we just initialize the client. Client w
 
 **(2)** Client configuration for the first cluster to be connected. Note that first `HazelcastOptionsBuilder` is able to configure to everything on the client, such as `HeartBeat`, `LoadBalancer` etc. This part is still `HazelcastOptions`. See details for client options [Options](configuration/options.md).
 
-**(3)** Alternative cluster options. Alternative options are only for connection and authentication. Alternative client options cannot override other fields. General configuration should be done in first client options which is step **1**. For example, here **3**, we cannot change the `SmartRouting` for each cluster. Also, for server side, provided cluster must have **same partition count**. Client cannot failover to a cluster with different partition count.
+**(3)** Alternative cluster options. Alternative options are only for connection and authentication. Alternative client options cannot override other fields. General configuration should be done in first client options which is step **1**. For example, here **3**, we cannot change the `~~~~RoutingMode` for each cluster. Also, for server side, provided cluster must have **same partition count**. Client cannot failover to a cluster with different partition count.
```

Comment:
```suggestion
**(3)** Alternative cluster options. Alternative options are only for connection and authentication. Alternative client options cannot override other fields. General configuration should be done in first client options which is step **1**. For example, here **3**, we cannot change the `~~~~RoutingMode` for each cluster. Also, for server side, provided cluster must have **same partition count**. Client cannot failover to a cluster with different partition count.
```
```suggestion
**(3)** Alternative cluster options. Alternative options are only for connection and authentication. Alternative client options cannot override other fields. General configuration should be done in first client options which is step **1**. For example, here **3**, we cannot change the `RoutingMode` for each cluster. Also, for server side, provided cluster must have **same partition count**. Client cannot failover to a cluster with different partition count.
```
Not sure what purpose the `~~~~` is serving here?

Reply:
No idea on where it came from. Good catch. Fixed